Assets/Tests/PlayModeTests/ConfigurationTests.cs

```diff
@@ -34,6 +34,12 @@ public void TestSDKInitParams()
                 EventQueueThreshold = 150,
                 TotalBreadcrumbsAllowed = 200,
 
+                MaxValueSize = 5,
```

Comment:
In these tests have a different value for each size. This way you can make sure that you are setting the correct variable